lib/Common/Memory/RecyclerPointers.h

```diff
@@ -375,6 +375,22 @@ class WriteBarrierPtr
         return result;
     }
 
+    WriteBarrierPtr& operator--()  // prefix --
+    {
+        --ptr;
+#ifdef RECYCLER_WRITE_BARRIER
+        RecyclerWriteBarrierManager::WriteBarrier(this);
+#endif
+        return *this;
+    }
+
+    WriteBarrierPtr operator--(int)  // postfix --
```

Comment:
Just to confirm, you added these operators for some other usages, not because of quick sort code added here right? Quicksort here only operates on `T*`, not `WriteBarrierPtr`.